sky/catalog/data_fetchers/fetch_runpod.py

```diff
@@ -0,0 +1,466 @@
+        'max_count': 8
+    },
+    'A100-80GB-SXM': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'A30': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'A40': {
+        'vcpus': 9.0,
+        'max_count': 10
+    },
+    'B200': {
+        'vcpus': 36.0,
+        'max_count': 8
+    },
+    'H100': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'H100-NVL': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'H100-SXM': {
+        'vcpus': 20.0,
+        'max_count': 8
+    },
+    'H200-SXM': {
+        'vcpus': 20.0,
+        'max_count': 8
+    },
+    'L4': {
+        'vcpus': 4.0,
+        'max_count': 8
+    },
+    'L40': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'L40S': {
+        'vcpus': 16,
+        'max_count': 8
+    },
+    'MI300X': {
+        'vcpus': 24.0,
+        'max_count': 8
+    },
+    'RTX2000-Ada': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTX3070': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3080': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3080-Ti': {
+        'vcpus': 8.0,
+        'max_count': 8
+    },
+    'RTX3090': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'RTX3090-Ti': {
+        'vcpus': 12.0,
+        'max_count': 8
+    },
+    'RTX4000-Ada': {
+        'vcpus': 9.0,
+        'max_count': 8
+    },
+    'RTX4090': {
+        'vcpus': 16.0,
+        'max_count': 8
+    },
+    'RTX5000-Ada': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTX5080': {
+        'vcpus': 5.0,
+        'max_count': 8
+    },
+    'RTX5090': {
+        'vcpus': 5.0,
+        'max_count': 8
+    },
+    'RTX6000-Ada': {
+        'vcpus': 14.0,
+        'max_count': 8
+    },
+    'RTXA4000': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTXA4500': {
+        'vcpus': 12.0,
+        'max_count': 8
+    },
+    'RTXA5000': {
+        'vcpus': 6.0,
+        'max_count': 8
+    },
+    'RTXA6000': {
+        'vcpus': 4.0,
+        'max_count': 8
+    },
+    'RTXPRO6000': {
+        'vcpus': 14.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-MaxQ': {
+        'vcpus': 18.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-WK': {
+        'vcpus': 32.0,
+        'max_count': 8
+    },
+    'V100-SXM2-32GB': {
+        'vcpus': 20.0,
+        'max_count': 8
+    }
+}
+
+# for backwards compatibility, force rename some gpus.
+# map the generated name to the original name
+GPU_NAME_OVERRIDES = {
+    'A100-PCIe': 'A100-80GB',
+    'A100-SXM': 'A100-80GB-SXM',
+    'H100-PCIe': 'H100',
+}
+
+# Constants
+USEFUL_COLUMNS = [
+    'InstanceType',
+    'AcceleratorName',
+    'AcceleratorCount',
+    'vCPUs',
+    'MemoryGiB',
+    'GpuInfo',
+    'Region',
+    'SpotPrice',
+    'Price',
+    'AvailabilityZone',
+]
+
+# Mapping of regions to their availability zones
+REGION_ZONES = {
+    'CA': ['CA-MTL-1', 'CA-MTL-2', 'CA-MTL-3'],
```

Comment:
these will get out of date, I suspect you can source this information dynamically via the api or html parsing. Either way, it's going to get out of sync and need to be updated so an imperfect source may be better.

Reply:
Agrees, I don't like hard coding these; however, I don't know of any way to get this information dynamically. Do you know how we can do that?